Docker build not functioning.

Steps to reproduce on a m1 mac: 
```
git clone https://github.com/grafana/detect-angular-dashboards
cd detect-angular-dashboards
docker build -t detect-angular-dashboards .
```
Error encountered: 
```
Sending build context to Docker daemon  647.7kB
Step 1/13 : FROM golang:1.20.6-alpine3.18 AS build
 ---> c987a8e5b947
Step 2/13 : WORKDIR /app
 ---> Using cache
 ---> cde3e808f825
Step 3/13 : COPY go.mod .
 ---> Using cache
 ---> e6d9c21c707c
Step 4/13 : COPY go.sum .
 ---> Using cache
 ---> 69ac653fa609
Step 5/13 : RUN go mod download -x
 ---> Using cache
 ---> 0d132c8bd7f9
Step 6/13 : COPY . .
 ---> 85d0ac47f6ae
Step 7/13 : RUN CGO_ENABLED=0 go build -v
 ---> Running in 8d5d1995ca27
detector/detector.go:7:2: package slices is not in GOROOT (/usr/local/go/src/slices)
note: imported by a module that requires go 1.24
```

Full output of the build. 
```
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

Sending build context to Docker daemon  647.7kB
Step 1/13 : FROM golang:1.20.6-alpine3.18 AS build
1.20.6-alpine3.18: Pulling from library/golang
8c6d1654570f: Pull complete
5a768fa670f5: Pull complete
853813baecd1: Pull complete
99b4e48867a2: Pull complete
Digest: sha256:7839c9f01b5502d7cb5198b2c032857023424470b3e31ae46a8261ffca72912a
Status: Downloaded newer image for golang:1.20.6-alpine3.18
 ---> c987a8e5b947
Step 2/13 : WORKDIR /app
 ---> Running in 498ddcedbf4b
 ---> Removed intermediate container 498ddcedbf4b
 ---> cde3e808f825
Step 3/13 : COPY go.mod .
 ---> e6d9c21c707c
Step 4/13 : COPY go.sum .
 ---> 69ac653fa609
Step 5/13 : RUN go mod download -x
 ---> Running in edff5524dbfc
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.mod
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.mod
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.mod: 200 OK (0.068s)
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.mod
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.mod: 200 OK (0.206s)
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.mod: 200 OK (0.275s)
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.mod
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.mod
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.mod: 200 OK (0.012s)
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.mod: 200 OK (0.012s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.mod
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.mod
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.mod: 200 OK (0.013s)
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.mod: 200 OK (0.013s)
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.mod
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.mod
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.mod: 200 OK (0.013s)
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.mod: 200 OK (0.013s)
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.mod
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.mod
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.mod
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.mod: 200 OK (0.010s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.mod
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.mod: 200 OK (0.204s)
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.mod
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.mod: 200 OK (0.189s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.7.0.mod
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.mod: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.9.mod
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.7.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.5.mod
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.9.mod: 200 OK (0.009s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.26.0.mod
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.5.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.2.mod
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.26.0.mod: 200 OK (0.008s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.3.1.mod
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.2.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190603091049-60506f45cf65.mod
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.3.1.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.2.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190603091049-60506f45cf65.mod: 200 OK (0.010s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.0.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.2.mod: 200 OK (0.010s)
# get https://proxy.golang.org/github.com/bwesterb/go-ristretto/@v/v1.2.3.mod
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.3.1-0.20221117191849-2c476679df9a.mod
# get https://proxy.golang.org/github.com/bwesterb/go-ristretto/@v/v1.2.3.mod: 200 OK (0.018s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.3.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.3.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/gopkg.in/check.v1/@v/v0.0.0-20161208181325-20d25e280405.mod
# get https://proxy.golang.org/gopkg.in/check.v1/@v/v0.0.0-20161208181325-20d25e280405.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.8.0.mod: 200 OK (0.008s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.6.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.6.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.6.0.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.6.0.mod: 200 OK (0.012s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.8.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230217124315-7d5c6f04bbb8.mod
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230217124315-7d5c6f04bbb8.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.8.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.2.mod
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.2.mod: 200 OK (0.012s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.10.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.10.0.mod: 200 OK (0.015s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.8.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.28.0.mod
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.28.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/xerrors/@v/v0.0.0-20191204190536-9bdfabe68543.mod
# get https://proxy.golang.org/golang.org/x/xerrors/@v/v0.0.0-20191204190536-9bdfabe68543.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.0.0-20190308221718-c2843e01d9a2.mod
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.0.0-20190308221718-c2843e01d9a2.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20190215142949-d0b11bdaac8a.mod
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.3.1-0.20221117191849-2c476679df9a.mod: 200 OK (0.194s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20190215142949-d0b11bdaac8a.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.0.0-20180917221912-90fa682c2a6e.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.26.0-rc.1.mod
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.0.0-20180917221912-90fa682c2a6e.mod: 200 OK (0.018s)
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.6.0.mod
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.26.0-rc.1.mod: 200 OK (0.017s)
# get https://proxy.golang.org/golang.org/x/mod/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.6.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.5.0.mod
# get https://proxy.golang.org/golang.org/x/mod/@v/v0.8.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.1.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.5.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.0.0-20210921155107-089bfa567519.mod
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.1.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/cloud.google.com/go/compute/metadata/@v/v0.2.0.mod
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.0.0-20210921155107-089bfa567519.mod: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.8.mod
# get https://proxy.golang.org/cloud.google.com/go/compute/metadata/@v/v0.2.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.8.0.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.8.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.8.mod: 200 OK (0.013s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.9.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.2.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.2.0.mod: 200 OK (0.013s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.2.0.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.9.0.mod: 200 OK (0.014s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.2.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.2.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.4.0.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.2.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/yuin/goldmark/@v/v1.4.13.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.4.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.6.0.mod
# get https://proxy.golang.org/github.com/yuin/goldmark/@v/v1.4.13.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.1.0.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.6.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.1.12.mod
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.1.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/bwesterb/go-ristretto/@v/v1.2.0.mod
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.1.12.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20211007075335-d3039528d8ac.mod
# get https://proxy.golang.org/github.com/bwesterb/go-ristretto/@v/v1.2.0.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20210226172049-e18ecbb05110.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20211007075335-d3039528d8ac.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20210615035016-665e8c7367d1.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20210226172049-e18ecbb05110.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.0.0-20201126162022-7de9c90e9dd1.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20210615035016-665e8c7367d1.mod: 200 OK (0.008s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.3.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.0.0-20201126162022-7de9c90e9dd1.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/mod/@v/v0.6.0-dev.0.20220419223038-86c51ed26bb4.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.3.mod: 200 OK (0.008s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20220722155257-8c9f86f7a55f.mod
# get https://proxy.golang.org/golang.org/x/mod/@v/v0.6.0-dev.0.20220419223038-86c51ed26bb4.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.5.0.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20220722155257-8c9f86f7a55f.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.7.0.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.5.0.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20220722155237-a158d28d115b.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.7.0.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.0.0-20220722155255-886fb9371eb4.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20220722155237-a158d28d115b.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.7.mod
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.0.0-20220722155255-886fb9371eb4.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20201119102817-f84b799fce68.mod
# get https://proxy.golang.org/golang.org/x/text/@v/v0.3.7.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.0.0-20191119224855-298f0cb1881e.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20201119102817-f84b799fce68.mod: 200 OK (0.008s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20220520151302-bc2c85ada10a.mod
# get https://proxy.golang.org/golang.org/x/tools/@v/v0.0.0-20191119224855-298f0cb1881e.mod: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/term/@v/v0.0.0-20210927222741-03fcf44c2211.mod
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.0.0-20220520151302-bc2c85ada10a.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190620200207-3b0461eec859.mod
# get https://proxy.golang.org/golang.org/x/term/@v/v0.0.0-20210927222741-03fcf44c2211.mod: 200 OK (0.010s)
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.0.0-20190423024810-112230192c58.mod
# get https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190620200207-3b0461eec859.mod: 200 OK (0.008s)
# get https://proxy.golang.org/golang.org/x/xerrors/@v/v0.0.0-20190717185122-a985d3407aa7.mod
# get https://proxy.golang.org/golang.org/x/sync/@v/v0.0.0-20190423024810-112230192c58.mod: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/xerrors/@v/v0.0.0-20190717185122-a985d3407aa7.mod: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.info
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.info: 200 OK (0.185s)
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.info
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.info: 200 OK (0.023s)
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.info
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.info: 200 OK (0.011s)
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.info
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.info: 200 OK (0.008s)
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.info
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.info: 200 OK (0.008s)
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.info
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.info: 200 OK (0.008s)
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.info
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.info: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.info
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.info: 200 OK (0.015s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.info
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.info: 200 OK (0.016s)
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.info
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.info: 200 OK (0.010s)
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.info
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.info: 200 OK (0.011s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.info
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.info: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.info
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.info: 200 OK (0.190s)
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.info
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.info: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.info
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.info: 200 OK (0.020s)
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.info
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.info: 200 OK (0.010s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.info
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.info: 200 OK (0.013s)
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.info
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.info: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.zip
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.zip
# get https://proxy.golang.org/github.com/google/go-github/v53/@v/v53.2.0.zip: 200 OK (0.019s)
# get https://proxy.golang.org/github.com/bradleyfalzon/ghinstallation/v2/@v/v2.6.0.zip: 200 OK (0.129s)
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.zip
# get https://proxy.golang.org/github.com/magefile/mage/@v/v1.15.0.zip: 200 OK (0.012s)
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.zip
# get https://proxy.golang.org/github.com/stretchr/testify/@v/v1.9.0.zip: 200 OK (0.264s)
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.zip
# get https://proxy.golang.org/github.com/%21proton%21mail/go-crypto/@v/v0.0.0-20230717121422-5aa5874ade95.zip: 200 OK (0.340s)
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.zip
# get https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.3.zip: 200 OK (0.014s)
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.zip
# get https://proxy.golang.org/github.com/davecgh/go-spew/@v/v1.1.1.zip: 200 OK (0.234s)
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.zip
# get https://proxy.golang.org/github.com/golang-jwt/jwt/v4/@v/v4.5.0.zip: 200 OK (0.082s)
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.zip
# get https://proxy.golang.org/github.com/golang/protobuf/@v/v1.5.3.zip: 200 OK (0.009s)
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.zip
# get https://proxy.golang.org/github.com/google/go-querystring/@v/v1.1.0.zip: 200 OK (0.010s)
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.zip
# get https://proxy.golang.org/github.com/pmezard/go-difflib/@v/v1.0.0.zip: 200 OK (0.009s)
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.zip
# get https://proxy.golang.org/golang.org/x/crypto/@v/v0.11.0.zip: 200 OK (0.035s)
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.zip
# get https://proxy.golang.org/golang.org/x/net/@v/v0.13.0.zip: 200 OK (0.144s)
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.zip
# get https://proxy.golang.org/golang.org/x/oauth2/@v/v0.10.0.zip: 200 OK (0.013s)
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.zip
# get https://proxy.golang.org/golang.org/x/sys/@v/v0.10.0.zip: 200 OK (0.061s)
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.zip
# get https://proxy.golang.org/google.golang.org/appengine/@v/v1.6.7.zip: 200 OK (0.013s)
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.zip
# get https://proxy.golang.org/google.golang.org/protobuf/@v/v1.31.0.zip: 200 OK (0.049s)
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.zip
# get https://proxy.golang.org/gopkg.in/yaml.v3/@v/v3.0.1.zip: 200 OK (0.408s)
 ---> Removed intermediate container edff5524dbfc
 ---> 0d132c8bd7f9
Step 6/13 : COPY . .
 ---> f037996017de
Step 7/13 : RUN CGO_ENABLED=0 go build -v
 ---> Running in cb53bbeeec69
detector/detector.go:7:2: package slices is not in GOROOT (/usr/local/go/src/slices)
note: imported by a module that requires go 1.24
The command '/bin/sh -c CGO_ENABLED=0 go build -v' returned a non-zero code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.20.6-alpine3.18 AS build
+FROM golang:1.24-alpine3.21 AS build
 
 WORKDIR /app
 
@@ -10,7 +10,7 @@ RUN go mod download -x
 COPY . .
 RUN CGO_ENABLED=0 go build -v
 
-FROM alpine:latest
+FROM alpine:3.21
 
 RUN apk add --no-cache ca-certificates
 

